packages/crypto/Cargo.toml

```diff
@@ -10,14 +10,16 @@ description = "Cryptographic tools for writing Secret Contracts"
 categories = ["cryptography::cryptocurrencies", "wasm"]
 keywords = ["secret-network", "secret-contracts", "secret-toolkit"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+hash = ["sha2"]
+rand = ["rand_chacha", "rand_core"]
+ecc = ["secp256k1"]
```

Comment:
I think we should add a
```toml
default = ["hash", "rand", "ecc"]
```

Reply:
Also I'd name the `ecc` feature more specifically, in case we want to add another ECC algorithm